content/tutorials/tags.json

```diff
@@ -0,0 +1,174 @@
+{
+  "Type": [
+    {
+      "value": "text",
+      "label": "Text Tutorial",
+      "description": "Tutorial in written format",
+      "color": "#010067"
+    },
+    {
+      "value": "video",
+      "label": "Video Tutorial",
+      "description": "Tutorial in video format",
+      "color": "#010067"
+    }
+  ],
+  "Topic": [
+    {
+      "value": "getting-started",
+      "label": "Getting Started",
+      "description": "Easy Tutorials for getting started",
+      "color": "#9e008e"
+    },
+    {
+      "value": "integration-services",
+      "label": "Integration Services",
+      "description": "Integrate IOTA seamlessly",
+      "color": "#0e4ca1"
+    },
+    {
+      "value": "hornet",
+      "label": "Hornet",
+      "description": "Hornet node software",
+      "color": "#0e4ca1"
+    },
+    {
+      "value": "bee",
+      "label": "Bee",
+      "description": "Bee node software",
+      "color": "#0e4ca1"
+    },
+    {
+      "value": "wasp",
+      "label": "Wasp",
+      "description": "Wasp smart contracts node software",
+      "color": "#0e4ca1"
+    },
+    {
+      "value": "chronicle",
+      "label": "Chronicle",
+      "description": "Chronicle permanode software",
+      "color": "#0e4ca1"
+    },
+    {
+      "value": "live-coding",
+      "label": "Live Coding",
+      "description": "Recorded live coding session",
+      "color": "#ffe502"
+    },
```

Comment:
Should we move this to the Type category?

Reply:
Yeah, makes sense

Reply:
Moved